services/src/main/java/org/exoplatform/addons/gamification/service/setting/rule/impl/RuleRegistryImpl.java

```diff
@@ -109,15 +109,7 @@ private void store(RuleConfig ruleConfig, RuleDTO ruleDTO) {
           ruleDto.setEvent(ruleConfig.getEvent());
           ruleDto.setLastModifiedBy("Gamification");
           ruleDto.setCreatedBy("Gamification");
-          ruleDto.setArea(ruleConfig.getZone());
```

Comment:
we may need to this in the future, should we test on null value of zone only instead of deleting the init behavior?

Reply:
Done!